server/src/concepts/user/controllers/index.js

```diff
@@ -1,18 +1,43 @@
 import validateCreateUser from "../model/UserValidation.js";
 import { createNewUser } from "../useCases/createNewUser.js";
+import { updateUserFunc } from "../useCases/updateUser.js";
 import { deleteUserWithId } from "../useCases/deleteUser.js";
 
 export const createUser = async (req, res) => {
   const validationResult = validateCreateUser(req.body);
   if (validationResult.error) {
-    return res.status(400).send("Invalid credentials");
+    return res.status(400).send("Invalid data");
```

Comment:
Next step can be to show which data is invalid. We have this informationin validationResult.error object